zabbix-smartmontools.conf

```diff
@@ -0,0 +1,40 @@
+[settings]
+# 'device' or 'serial' as primary identifier in zabbix item's name
+# 'serial' is preferred for multi-disk systems
+# mode: device
+
+# skip duplicate disk outputs. 'DriveStatus' json will not be skipped
+# determined by disk serial, model, capacity and firmware (serial + at least
+# one of others)
+# skipDuplicates: yes
+
+# Path to the smartctl binary
+# ctlPath: smartctl
+# On Windows:
+# ctlPath = C:\Program Files\smartmontools\bin\smartctl.exe
+# On FreeBSD
+# ctlPath = '/usr/local/sbin/smartctl
+
+# path to second send script
+# senderPyPath: /etc/zabbix/scripts/sender_wrapper.py              # Linux
```

Comment:
System hint comment is making setting like that to fail. We still need the hint.

Reply:
I don't understand.  Could you please explain?

Reply:
This fails:
`senderPyPath: /etc/zabbix/scripts/sender_wrapper.py              # Linux`
This don't:
`senderPyPath: /etc/zabbix/scripts/sender_wrapper.py`